Core/Core/Syllabus/SyllabusActionableItemsCell.xib

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="15505" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <device id="retina6_1" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="15510"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="SyllabusActionableItemsCell" customModule="Core" customModuleProvider="target">
+            <connections>
+                <outlet property="dateLabel" destination="sBO-lg-S52" id="zDC-Lt-9ET"/>
+                <outlet property="iconImageView" destination="fUC-Pd-E2a" id="nuL-lC-Lsb"/>
+                <outlet property="itemNameLabel" destination="T12-iA-cYO" id="iuW-L3-EYF"/>
+                <outlet property="stackView" destination="EaW-6y-Jo2" id="UsN-fq-j6Q"/>
+            </connections>
+        </placeholder>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleToFill" id="iN0-l3-epB">
+            <rect key="frame" x="0.0" y="0.0" width="414" height="100"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="4" translatesAutoresizingMaskIntoConstraints="NO" id="EaW-6y-Jo2">
```

Comment:
I think the spacing should be 2. It is for Grades. It looks too spread out to me. Can you double check with Dave?

Reply:
this is how he wanted it.  I put it at 2 and he thought it was too close

Reply:
it changes daily

Reply:
ah I see